### PR TITLE
Update incident-response-plan.md

### PR DESCRIPTION
**Pull request by dmundra**
[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/dmundra-incident-response-plan-patch/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

**Comment by dmundra** [comment]
I see the check that the anchor link changes I made are not valid. The generated links don't appear to be working on this page https://handbook.civicactions.com/en/latest/100-security/incident-response-plan/, is there an alternate change I can make or just revert the changes for now?

File: docs/100-security/incident-response-plan.md [hunk 1]
```diff
@@ -13,9 +13,9 @@
     - [Comms at the Remediate phase](#comms-at-the-remediate-phase)
   - [Retrospective](#retrospective)
 - [Incident Severities](#incident-severities)
-  - [1 - High Severity](#1---high-severity)
-  - [2 - Medium Severity](#2---medium-severity)
-  - [3 - Low Severity](#3---low-severity)
+  - [1 - High Severity](#1-high-severity)
+  - [2 - Medium Severity](#2-medium-severity)
+  - [3 - Low Severity](#3-low-severity)
```
[the remark this answers is not in the document]
Comment (openprivacy) [reply docs/100-security/incident-response-plan.md]: It appears that your changes (e.g., `3-low-severity`) should work but that the link checker isn't allowing them due to the `space-dash-space` in the heading. Maybe get rid of the numbers in the headings altogether?

Thanks for taking the time to look at this. Feel free to revert if too much hassle.

[the remark this answers is not in the document]
Comment (dmundra) [reply docs/100-security/incident-response-plan.md]: Reverted for now. It appears that even in GitHub preview the change of link doesn't work.

**Review by openprivacy — APPROVED** [review]
Thank you. And note that we will be updating this soon with new versions derived from the SSP-Toolkit <https://github.com/CivicActions/ssp-toolkit/blob/master/appendices/security-irp.md>

**Pull request merged**